stack.yaml

```diff
@@ -77,14 +76,20 @@ extra-deps:
 - git: https://github.com/serokell/temporary.git
   commit: 102ed4671dd78c5d271f09756681aac2a41b6717
 
+# Fixes missing resources cleanup in case of async exception.
```

Comment:
Very nice!

Reply:
Yeah, do the PR - this is also a way of knowing whether or not there were some deep idea behind not handling async exceptions (seems really unlikely, but anyway).